gatsby_fp/src/styles/facultyprofile.css

```diff
@@ -1,14 +1,23 @@
 /* Styles for Faculty Profile */
 
 body {
-  font-size: 1.1em;
+  font-family: "Chaparral W01", Georgia, Times, "Times New Roman", serif;
+  font-style: normal;
+  font-weight: 300;
+  min-width: 300px;
+  overflow-x: hidden;
 }
 
 .sectiontitle {
-  font-size: 1.4em;
-  font-weight: bold;
-  padding-top: 0.4em;
-  margin-bottom:-0.2em;
+  font-family: "Trade Gothic W01", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 1.875rem;
+  font-size: 30px;
```

Comment:
I don't think there's a need to have `font-size` twice. For accessibility, we have standardized on avoiding `px`. I'd recommend comparing how this looks at 100% viewport scale, with and without this `30px` enabled, and adjust the `1.875` rem as needed so that you can eliminate the `30px` without changing the typical appearance. @davidwatson agree?

Reply:
I removed this and it had virtually no effect on the size in my dev build